kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/SmokeTests.java

```diff
@@ -3,12 +3,12 @@
 import com.provectus.kafka.ui.base.BaseTest;
 import io.qameta.allure.Issue;
 import lombok.SneakyThrows;
-import org.junit.Ignore;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 @Disabled // TODO #1480
+@SuppressWarnings("squid:S2699")
```

Comment:
?

Reply:
Sonar believes that a class with a name containing "Test" must contain tests. I'll see what I can do to make it look more elegant

Reply:
Removed @SuppressWarnings. Seems like we need to fix these tests first